content/blog/2019/announcing-1.0-eol/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Support for Istio 1.0 ends on June 19th, 2019
+description: Upcoming Istio 1.0 EOL announcement
```

Comment:
Is 'EOL' the right word here?

Reply:
I don't think EOL is a well-known acronym. How about just saying "end-of-life"